packages/kbn-securitysolution-io-ts-alerting-types/src/actions/index.ts

```diff
@@ -6,42 +6,47 @@
  * Side Public License, v 1.
  */
 
-/* eslint-disable @typescript-eslint/naming-convention */
-
 import * as t from 'io-ts';
 import { saved_object_attributes } from '../saved_object_attributes';
 
+export type RuleActionGroup = t.TypeOf<typeof RuleActionGroup>;
+export const RuleActionGroup = t.string;
```

Comment:
@marshallmain This is a TypeScript feature called aliases. In this case, it's handy because it allows to write only one JSDoc comment which will be applied to both the TS type and the io-ts schema. So when you hover over them in the IDE you will see the same comment. Without that, we'd need to either duplicate the comment or leave either the TS type or the io-ts schema uncommented. Personally, I also like the fact that it's 1 import instead of 2 when you need both (e.g. in routes).

Reply:
One of the examples of how JSDoc comments for rule attributes could look like: `x-pack/plugins/security_solution/common/detection_engine/rule_schema/model/common_attributes/related_integrations.ts`